README.md

```diff
@@ -131,6 +133,38 @@ func main() {
 }
 ```
 
+### De-duplication of assignments
```

Comment:
This documentation was missing; the feature has already been released.

Reply:
🙌